runtime/int.go

```diff
@@ -384,6 +400,7 @@ func initIntType(dict map[string]*Object) {
 	IntType.slots.Div = &binaryOpSlot{intDiv}
 	IntType.slots.DivMod = &binaryOpSlot{intDivMod}
 	IntType.slots.Eq = &binaryOpSlot{intEq}
+	IntType.slots.FloorDiv = &binaryOpSlot{intFloorDiv}
```

Comment:
Thinking about it a little more, I think you can just use intDiv for this slot. I think it does exactly what you want.